src/app/users/users.component.ts

```diff
@@ -112,7 +112,12 @@ export class UsersComponent implements OnInit, AfterViewInit {
   }
 
   getUsers() {
-    return this.couchService.post(this.dbName + '/_find', { 'selector': {} });
+    return this.couchService.allDocs(this.dbName)

```

Comment:
non admin will not be able to get alldocs that's why we have introduced _find so this will not work